src/features/ProtectTransaction/components/ProtectTxProtection.tsx

```diff
@@ -131,11 +134,11 @@ const Header = styled.h4`
 `;
 
 interface Props {
-  sendAssetsValues: IFormikFields | null;
+  overrideValues?: IFormikFields;
```

Comment:
This is surprising.
If the parent knows that it needs to 'override' why wouldn't just pass the correct values ini the first place?

Reply:
This is exclusively used for the stories at this point. I.e. `ProtectTxProtection.stories.tsx`